tests/foreman/cli/test_import.py

```diff
@@ -1,12 +1,20 @@
 # -*- encoding: utf-8 -*-
 """Test class for Host Collection CLI"""
+import csv
+import tempfile
 from ddt import ddt
 from fauxfactory import gen_string
-from os.path import join
+from os import remove
```

Comment:
I would import os directly and use `os.remove`, `os.path.join` and `os.path.basename` on code.